fractal-puppy/Engine
# Improved temporary scene save and load

**Before:** The temporary scene was saved on Assets/Scenes and imported to Library/Scenes. It was treated as any other scene created by the user. The temporary scene wasn't deleted after load.
**Bug:** If you pressed "play" and quickly pressed "stop" the engine wasn't fast enough to import the file, so it loaded an old temporary scene.

**Now:** The temporary scene is saved on Resources/Scenes and treated as an engine resource, so it doesn't need to be imported. After loading the temporary scene it is deleted.

**How to test:**
Test 1:
- Start the engine and create a scene or load any previously created scene.
- Press play and change some Game Objects of the scene.
- Press stop and see if the scene has been set back to how it was before pressing play.

Test 2:
- Start the engine and create a scene or load any previously created scene.
- Press play and try to press stop as soon as you can.
- See if the scene loaded is the same you already had.

Example:
https://gyazo.com/65c861379d50ca8a2c5873302beb054d
